contracts/Controllable.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Controllable is Ownable {
```

Comment:
If you inherit Ownable here as well, is this the same Ownable as on the main contract or are they separate? It's pretty confusing having multiple Ownables to me, but not sure if this is standard practice

Reply:
It's the same one - Solidity 'linearises' the parent classes.